apps/s_socket.c

```diff
@@ -309,6 +310,9 @@ int do_server(int *accept_sock, const char *host, const char *port,
             ERR_print_errors(bio_err);
             goto end;
         }
+    } else {
+        (void)BIO_printf(bio_s_out, "ACCEPT\n");
+        (void)BIO_flush(bio_s_out);
     }
```

Comment:
Side note: Since both branches of the condition do `BIO_flush`, an option would be to bring it down below...

Reply:
Oh, wait, never mind.